go/vt/servenv/grpc_server_auth_static.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2017 Google Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package servenv
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+	"io/ioutil"
+
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/metadata"
+
+	log "github.com/golang/glog"
+)
+
+var (
+	credsFile = flag.String("grpc_server_auth_static_file", "", "JSON File to read the users/passwords from.")
+	// StaticAuthPluginConfig implements AuthPlugin interface
```

Comment:
I'd rename the struct to `StaticAuthPlugin` and change the comment to say
"StaticAuthPlugin implements the AuthPlugin interface".